products/monitoring/api.yaml

```diff
@@ -0,0 +1,785 @@
+                  should be compared with the threshold.The
+                  filter is similar to the one that is
+                  specified in the
+                  MetricService.ListTimeSeries request (that
+                  call is useful to verify the time series
+                  that will be retrieved / processed) and must
+                  specify the metric type and optionally may
+                  contain restrictions on resource type,
+                  resource labels, and metric labels. This
+                  field may not exceed 2048 Unicode characters
+                  in length.
+           - !ruby/object:Api::Type::String
+             name: displayName
+             required: true
+             description: |
+               A short name or phrase used to identify the
+               condition in dashboards, notifications, and
+               incidents. To avoid confusion, don't use the same
+               display name for multiple conditions in the same
+               policy.
+        required: true
+      - !ruby/object:Api::Type::Array
+        name: 'notificationChannels'
+        description: |
+          Identifies the notification channels to which notifications should be
+          sent when incidents are opened or closed or when new violations occur
+          on an already opened incident. Each element of this array corresponds
+          to the name field in each of the NotificationChannel objects that are
+          returned from the notificationChannels.list method. The syntax of the
+          entries in this field is
+          `projects/[PROJECT_ID]/notificationChannels/[CHANNEL_ID]`
+        item_type: Api::Type::String
+      - !ruby/object:Api::Type::Array
+        name: 'labels'
+        description: 'User-supplied key/value data to be used for organizing AlertPolicy objects.'
+        item_type: Api::Type::String
+
+
+  - !ruby/object:Api::Resource
+    name: 'Group'
+    base_url: projects/{{project}}/groups
+    self_link: projects/{{project}}/groups/{{name}}
+    update_verb: :PATCH
+    input: true
+    description: |
+      The description of a dynamic collection of monitored resources. Each group
+      has a filter that is matched against monitored resources and their
+      associated metadata. If a group's filter matches an available monitored
+      resource, then that resource is a member of that group.
+    exports:
+      - !ruby/object:Api::Type::FetchedExternal
+        name: name
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/monitoring/docs/'
+      api: 'https://cloud.google.com/monitoring/api/ref_v3/rest/'
+    properties:
+      - !ruby/object:Api::Type::String
+        name: parentName
+        description: |
+          The name of the group's parent, if it has one. The format is
+          "projects/{project_id_or_number}/groups/{group_id}". For
```

Comment:
At this point I haven't done any work/testing on group it's mostly slurped up from hawk's commits so I'm going to wait to release it until I can put time towards it. Since I don't plan on releasing Group as part of this PR I'll add as a TODO for when I get Group finished.